### ISSUE TITLE
Student submitting zero-sum contribution question: the error message is misleading

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#11950]
v8.19.2 live server

Scenario: A student is submitting responses for a contribution question. The zero-sum constraint is in force. The team has five members. The student enters `Equal share + 20%` for all five members.

Error message shown:
![image](https://user-images.githubusercontent.com/1673303/186937423-8fd26233-861d-4756-9126-aa18f52e8664.png)

It should read `Current total 5 x Equal share + 100% ...` as the team has only 5 members.

**Comment by domlimm** [comment]
Hello prof, may I ask if this will be a good first issue? If not, I'd love to work on this 🙂

**Comment by damithc** [comment]
@domlimm let me mark it for first timers, for the time being. @xyliew25 is this suitable for a first timer (i.e., an easy fix)?

**Comment by damithc** [comment]
On second thoughts, @domlimm go ahead and fix it. I think it's best to fix this soon as it can result in unnecessary support requests.

**Comment by domlimm** [comment]
Will fix it ASAP. Thanks prof!

**Comment by domlimm** [comment]
Hello prof, I thought I should give you an update on this. I took a look and have a rough idea of what needs to be done. However, I need to test it and also fully understand and see how it works.

2 days ago, I deleted my containers and images from Docker desktop (no idea why I would do that 😓), and was unable to access the app the same night you made this issue.

I created an issue for my problem #11951, still unable to resolve it to date.
